spec/decorators/article_decorator_spec.rb

```diff
@@ -1,4 +1,56 @@
 require "rails_helper"
 
 RSpec.describe ArticleDecorator do
+  describe "#authors" do
+    subject { described_class.new(article) }
```

Comment:
Can you also move the subject after the `let`s. I very much follow Given, When, Then for basic structure on Orientation.